app/create_file.py

```diff
@@ -1 +1,48 @@
-# write your code here
+from datetime import datetime
+import sys
+import os
+
+command = sys.argv
+
+
+def create_directory() -> None:
+    index_d = command.index("-d") if "-d" in command else -1
+    index_f = command.index("-f") if "-f" in command else -1
```

Comment:
```suggestion
    index_d = command.index("-d") if "-d" in command else None
    index_f = command.index("-f") if "-f" in command else None
```
Better to use None instead of some magic numbers